Nef_S2/include/CGAL/Nef_S2/Sphere_circle.h

```diff
@@ -117,12 +123,12 @@ Plane_3 plane() const { return Base(*this); }
 Plane_3 plane_through(const Point_3& p) const
 /*{\Mop returns the plane parallel to |\Mvar| that
 contains point |p|.}*/
-{ return Plane_3(p,((Base*) this)->orthogonal_direction()); }
+{ return Plane_3(p,Base::orthogonal_vector()); }
```

Comment:
Is this change just a cleanup?

Reply:
Yes, I've added a comment to the change summary.

Reply:
Oh, wait. I changed `orthogonal_direction` to `orthogonal_vector`. I don't think it makes a difference, down the call chain it calls v.to_direction() anyway.